src/components/labelledInputTextField/labelledInputTextField.jsx

```diff
@@ -0,0 +1,36 @@
+import "./labelledInputTextField.scss";
+
+const LabelledInputTextField = ({
+  uniqueInputFieldHtmlName,
+  inputType,
+  labelHeader,
+  inputLarge,
+}) => {
+  let inputClassName = "labelled-input-text-field__input";
+
+  if (inputLarge) {
+    inputClassName = "labelled-input-text-field__input input-large";
```

Comment:
input-large is a modifier: `labelled-input-text-field__input--large` would be the additional classname... see why it's a good idea for a shorter component name?